wiki/Welcome/en.md

```diff
@@ -1,148 +1,37 @@
 
-Achievements are little badges on one's profile which indicate that someone has accomplished an exceptional goal. Achievements' badge can be unlocked in various ways such as playing a lot or hitting a certain combo count in one beatmap. When you fulfilled the criteria, you will receive a badge at "Online Results" and your account's personal achievement tab. However, the meat of it is the hidden mission, Hush-Hush, where the criteria is not known and it is up to us to figure it out on how to get it. They are pretty hard to get but it is fulfilling when you managed to achieved it by yourself.
+## Ranking process
 
-**Note:** Achievement badges can only be unlocked online based on your current account logged-in.
+_See also: [Beatmap Ranking Procedure](/wiki/Beatmap_Ranking_Procedure)._
 
-## Chat
-*Full explanation: [Chat Console][Chat_Console wikilink]*
+The ranking process of a beatmap (a difficulty that the player can play) consists of [beatmapping](/wiki/beatmapping/#getting-started) and [modding](/wiki/modding/#getting-started).
 
-Knowing the basics on how the [Chat Console][Chat_Console wikilink] works is important to those who are interested into making new friends in our friendly environment. It's also important to understand how to use it in case someone messages you (unlikely to happen unless you are well-known), telling you tips or asking for help themselves. There is a button to open up the chat on the bottom right of the screen but it's way quicker to just press F8 on your keyboard.
+Beatmapping is the process of a creator creates a beatmap. This process includes: selecting music, timing and mapping the beatmap, and testing the beatmap. It might also include: adding a video, adding a [storyboard](/wiki/storyboarding/#getting-started), and/or adding a [custom skin](/wiki/skinning/#getting-started). All of those can be done by the creator alone or with other users, sometimes referred to as _storyboarders_, _skinners_, and/or _guest creators_, depending on what they do.
 
-You will have a few tabs like "\#osu!", "\#announce", "\#userlog" or if you're in a conversation with someone, you'll see their name as a separate tab. Navigating through the tabs is extremely easy, just press on the tab and type your message then press "ENTER" to send the message to the opened tab. Regardless, please do not spam (continuously sending short messages into the chat) as doing so will automatically mute your account for a few minutes and your "hard-work" removed from the chat. You can press "F9" to open the user list (called Extended Chat Console) connected to the server, and you can apply filter to narrow down your search.
+Modding is the process of users review (or commonly called "mod") a creator's beatmap in the pending (or work in progress/help) stage. Modding plays a big role in quality control because this allows creators to fix issues with their beatmap.
 
-![Chat Console friendlist][Chat_Console friendlist image]
+## osu!wiki - How you can help!
```

Comment:
## How can you help!

this should be about whole osu!, not just osu! (link to the proper article too)

Reply:
See #94 
> A contributor section dedicated to linking to this repository and introducing users to how they might contribute (it is a wiki, after all)

That's what happened